app/(tabs)/_layout.tsx

```diff
@@ -17,43 +21,250 @@ function TabBarIcon(props: {
-                    style={{ marginRight: 15, opacity: pressed ? 0.5 : 1 }}
-                  />
-                )}
-              </Pressable>
-            </Link>
-          ),
-        }}
-      />
-      <Tabs.Screen
-        name="two"
-        options={{
-          title: 'Tab Two',
-          tabBarIcon: ({ color }) => <TabBarIcon name="code" color={color} />,
-        }}
-      />
-    </Tabs>
+    <ThemeProvider>
+      <Tabs
+        screenOptions={{
+          tabBarActiveTintColor: Colors[colorScheme ?? 'light'].tint,
+          // Disable the static render of the header on web
+          // to prevent a hydration error in React Navigation v6.
+          headerShown: useClientOnlyValue(false, true),
+          tabBarStyle: {
+            shadowColor: "gray",
+            shadowOffset: {
+                width: 10,
+                height: 10,
+            },
+            shadowOpacity: 1,
+            shadowRadius: 10,
+            },
+            headerTitleStyle: {
+                fontSize: 40,
+                fontWeight: "normal",
+                marginLeft: "3%",
+            },
+            headerStyle: {
+                backgroundColor: "#f2f2f2",
+                shadowColor: "#f2f2f2",
+                shadowOffset: {
+                    width: 0,
+                    height: 15,
+                },
+                shadowOpacity: 100,
+                shadowRadius: 10,
+            },
+            headerTitleAlign: "left",
+        }}>
+
+        <Tabs.Screen
+          name="index"
+          options={{
+            tabBarIcon: ({ color }) => <TabBarIcon name="code" color={color} />,
+            headerRight: () => (
+              <Link href="/modal" asChild>
+                <Pressable>
+                  {({ pressed }) => (
+                    <FontAwesome
+                      name="info-circle"
+                      size={25}
+                      color={Colors[colorScheme ?? 'light'].text}
+                      style={{ marginRight: 15, opacity: pressed ? 0.5 : 1 }}
+                    />
+                  )}
+                </Pressable>
+              </Link>
+            ),
+          }}
+        />
+
+        <Tabs.Screen
+            name="lists"
+            options={{
+              headerShown: true,
+              headerShadowVisible: false,
+              headerTitleStyle: {
+                fontSize: 30,
+                fontWeight: "normal",              
+              },
+              headerStyle: {
+                  backgroundColor: "#f2f2f2",
+                  height: 120,
+                  justifyContent: 'flex-end',
+              },
+              tabBarShowLabel: false,
+              tabBarIcon: ({ focused }) => <TabBarIconTest focused={focused} icon="list" />,
+              headerRight: () => (
+                <HeaderButtons 
+                    buttons={[
+                        {
+                            icon: <AntDesign name="pluscircleo" size={35} color="gray" />,
```

Comment:
Use this icon instead:
<Ionicons name="add-circle-outline" size={40} color="gray" />